doc/doc/basics/extractors.md

```diff
@@ -17,10 +17,15 @@ class contains different extractor *styles*:
 
 - Row-wise extractors:
   - [`json_extractor`](#json_extractor): highly flexible and compatible with
-  previous systems, but with limited performance 
+  previous systems, but with limited performance
```

Comment:
Last time I tried reverting space changes, I ended up destroying my working copy...

Good news: Adding 'w=1' to the URL suppresses whitespace differences: https://github.com/tiimgreen/github-cheat-sheet#ignore-whitespace

Also, if we switch to Phabricator, we get smarter diff by default, together with lots of configurable linters.